### DISCUSSION TITLE
List of AWS services required by Hubs Cloud

### DISCUSSION THREAD

**Discussion by mohrtw** [mozilla/hubs#3035] [Hubs Cloud Questions] [answered]
Could we get a list of all the AWS services required to deploy Hubs Cloud in the documentation?

**Comment by robinkwilson** [comment, marked as answer]
Good question! 

From the Hubs docs: [Hubs Cloud AWS architecture](https://hubs.mozilla.com/docs/hubs-cloud-aws-architecture.html)

**Reply by mohrtw** [reply]
Thanks, so is this the complete list?

CloudFront
API Gateway
AWS Lambda
Elastic Load Balancing (ELB)
AWS Certificate Manager
AWS Secrets Manager
AWS Budgets
Elastic File System (EFS)
Aurora Serverless - PostgreSQL
Simple Storage Servcie (S3)
Virtual Private Cloud (VPC)
EC2 Auto Scaling
Relational Database Service (RDS)
(optional) Route53
(optional) Simple Email Service (SES)

**Reply by mohrtw** [reply]
I ask because from the discussion about HC deployments in China (#3034) the problem was that AWS China was missing a service. However, all of these services besides the optional ones (Route53 and SES) [are available in AWS China](https://www.amazonaws.cn/en/about-aws/regional-product-services/). Route53 is even available in Ningxia. Am I missing something?

**Reply by mohrtw** [reply]
Mumbai, Seoul, Singapore, and Sydney [have all of these services, even Route53 and SES](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/). Yet the HC CloudFormation doesn't give these regions as an option to deploy.

**Reply by ocularvr** [reply]
Would be so nice if Mozilla had a division that allowed for projects like Hubs to be deployed by users. Right now, Mozilla is making a lot of money for amazon and things like the AWS awful technical support (you have to pay for this also!!!!) does not do any justice to products like Hubs. If Mozilla told me to pay 500 USD a year to deploy a basic solution for up to 24 users, I would pay Mozilla happily - Just let the people deploy their own instance. Really, you guys should not be laying off people and making money for Amazon. Think about it. I love Mozilla so count on customers like me for payment.
